nixos/doc/manual/administration/cleaning-store.xml

```diff
@@ -14,14 +14,14 @@ collector</emphasis> to remove old, unreferenced packages.  This is
 easy:
 
 <screen>
-$ nix-collect-garbage
+# nix-collect-garbage
```

Comment:
I can't find any mention of the nix daemon in the manual itself (maybe I'm missing something), so I'd leave it for now and mark it as requiring root and maybe change it later when documentation of the daemon is added.